packages/frontend2/src/app/(new)/(other)/scaling/tvl/_components/table/columns.tsx

```diff
@@ -0,0 +1,157 @@
+import { createColumnHelper } from '@tanstack/react-table'
+import Image from 'next/image'
+import { UpcomingBadge } from '~/app/_components/badge/upcoming-badge'
+import { IndexCell } from '~/app/_components/table/cells/index-cell'
+import { ProjectNameCell } from '~/app/_components/table/cells/project-name-cell'
+import { cn } from '~/utils/cn'
+import { type ScalingTvlTableRow } from '../../_utils/to-table-rows'
+import { TotalValueLockedCell } from './total-value-locked-cell'
+import { ValueLockedCell } from './value-locked-cell'
+
+const columnHelper = createColumnHelper<ScalingTvlTableRow>()
+
+const totalColumn = columnHelper.accessor('tvl', {
+  id: 'total',
+  header: 'Total',
+  cell: (ctx) => {
+    const value = ctx.getValue()
+    if (!value.breakdown) {
+      return <UpcomingBadge />
+    }
+    return (
+      <TotalValueLockedCell
+        tvlWarnings={value.warnings}
+        breakdown={{
+          canonical: value.breakdown.canonical,
+          external: value.breakdown.external,
+          native: value.breakdown.native,
+        }}
+        change={value.totalChange}
+      />
+    )
+  },
+  sortUndefined: 'last',
+  meta: {
+    align: 'center',
+    tooltip: 'Total = Canonical + External + Native',
+  },
+})
+
+export const scalingTvlCokumns = [
+  columnHelper.accessor((_, index) => index + 1, {
+    header: '#',
+    cell: (ctx) => <IndexCell>{ctx.row.index + 1}</IndexCell>,
+    meta: {
+      headClassName: 'w-0',
+    },
+  }),
+  columnHelper.display({
+    id: 'logo',
+    cell: (ctx) => (
+      <Image
+        className="min-h-[18px] min-w-[18px]"
+        src={`/icons/${ctx.row.original.slug}.png`}
+        width={18}
+        height={18}
+        alt={`${ctx.row.original.name} logo`}
+      />
+    ),
+    meta: {
+      headClassName: 'w-0',
+      cellClassName: '!pr-0',
+    },
+  }),
+  columnHelper.accessor('name', {
+    cell: (ctx) => <ProjectNameCell project={ctx.row.original} type="layer2" />,
+  }),
+  columnHelper.group({
+    id: 'data',
+    header: undefined,
+    columns: [totalColumn],
+  }),
+  columnHelper.accessor('tvl', {
+    id: 'canonical',
+    header: 'Canonical',
+    cell: (ctx) => {
+      const value = ctx.getValue()
+      if (!value.breakdown) {
+        return <UpcomingBadge />
+      }
+
+      return (
+        <ValueLockedCell
+          value={value.breakdown.canonical}
+          change={value.change?.canonical}
+          tokens={[]}
```

Comment:
if we decided we are not implementing the tokens tooltip then lets remove it from ValueLockedCell too